js/script.js

```diff
@@ -0,0 +1,41 @@
+import { $taskInput, $taskContainer, $addButton, $errorMessage} from "./element.js";
+import { sanitizeInput } from "./utility.js";
+
+let taskList = [];
+
+const addTaskHandler = () => {
+    const taskTitle = sanitizeInput($taskInput.value);
+    
+    if (!taskTitle.length) {
+        $errorMessage.hidden = false;
+        $errorMessage.innerHTML = "Task Name must be provided";
+        return;
+    }
+    else{
+        $errorMessage.hidden = true;
+    }
```

Comment:
As you are early returning the `else` block is redundant
```suggestion
    $errorMessage.hidden = true;
```